osbs/build/build_request.py

```diff
@@ -377,17 +377,34 @@ def adjust_for_registry_api_versions(self):
     def adjust_for_triggers(self):
         """
         Remove trigger-related plugins if no triggers set
+        import_image plugin is only removed for non-custom base images
```

Comment:
I'm wondering if it ever makes sense to turn off this plugin.

base -> imageA -> imageB

If imageA does not get automatically triggered, but imageB does, what will ever trigger imageB?